tests/rhui3_tests/test_client_management.py

```diff
@@ -56,7 +93,56 @@ def test_05_create_cli_rpm():
     Expect.enter(connection, 'q')
     Expect.ping_pong(connection, "test -f /root/test_cli_rpm-3.0/build/RPMS/noarch/test_cli_rpm-3.0-1.noarch.rpm && echo SUCCESS", "[^ ]SUCCESS")
 
-def test_06_create_docker_cli_rpm():
+def test_08_remove_rhui_conf_rpm():
+    '''
+       remove rhui configuration rpm from client
+    '''
+    Util.remove_conf_rpm(cli)
```

Comment:
Oh, wait, I think I spoke too early. The `/etc/yum/pluginconf.d/rhui-lb.conf` file comes from `rh-amazon-rhui-client` and provides the system with all those `rhui-REGION-*` repos.

Why are you calling `remove_conf_rpm()`? To get rid of the `rhui-REGION-*` repos or a client configuration RPM that might have been installed from previous test efforts? From the next methods in the test case, it seems to me you're making room for the newly created client configuration RPM, `test_cli_rpm-3.0-1.noarch.rpm`.

PS. `remove_conf_rpm()` should really be renamed to be more specific. It's not clear what kind of configuration this method removes.